### DISCUSSION TITLE
Observation: Local database update very slow

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#966] [Fixed Issues]
rAudio 1 • 202207081: 
An "Update database" of a single album lasts for a very long time (90 secs in my example). 
Can anybody confirm this?
Possible correlation to today's update of mpd to v0.23.8-1?

**Comment by rern** [comment]
Please try the UPDATE branch.

**Reply by klassikmann** [reply]
Fixed. Thank you.
